### PR TITLE
Setup GitHub Actions to generate native images

### PR DESCRIPTION
**Pull request by olafurpg**
This PR adds a new CI job to build a GraalVM native image for Linux and macOS. The `scalafmt` binary can be downloaded from the GitHub Actions UI under the "Artifacts" tab

Screenshot 2019-11-27 at 10 57 45

### CODE REVIEW

**Comment by olafurpg** [comment]
For releases, I'm planning to start with manually uploading the zip files. I suspect we don't do stable release too often so a manual approach should be fine for now.

File: scalafmt-cli/src/main/scala/org/scalafmt/cli/Cli.scala [hunk 1]
```diff
@@ -89,9 +89,21 @@ object Cli {
...
       case None => ScalafmtCoreRunner
-      case Some(v) if v == Versions.version =>
-        ScalafmtCoreRunner
-      case _ => ScalafmtDynamicRunner
+      case Some(v) =>
+        if (v == Versions.version) ScalafmtCoreRunner
+        else {
+          val isNativeImage =
+            "true" == System.getProperty("scalafmt.native-image", "false")
+          if (isNativeImage) {
```
Comment (poslegm) [inline scalafmt-cli/src/main/scala/org/scalafmt/cli/Cli.scala]: I like it 👍

Comment (olafurpg) [reply scalafmt-cli/src/main/scala/org/scalafmt/cli/Cli.scala]: This doesn't seem to work yet, I'll try to fix it before merging.

Comment (olafurpg) [reply scalafmt-cli/src/main/scala/org/scalafmt/cli/Cli.scala]: OK, should be fixed now!
Screenshot 2019-11-27 at 14 51 02

**Pull request merged**